src/MainWindow.vala

```diff
@@ -112,6 +113,7 @@ public class AppCenter.MainWindow : Hdy.ApplicationWindow {
         notify["working"].connect (() => {
             Idle.add (() => {
                 spinner.active = working;
+                refresh_menuitem.sensitive = !working;
```

Comment:
Instead of setting the menuitem sensitive, this should set the action as disabled so that we also disabled the keyboard shortcut